src/utils/is.js

```diff
@@ -20,7 +20,7 @@ export function isRegexp (v) {
 }
 
 export function isNumber (v) {
-  return typeof v === 'boolean'
+  return !isNaN(parseFloat(v)) && isFinite(v)
```

Comment:
This actually returns `true` when checking `'2`, which is wrong.
A better way would be: `return typeof v === 'number' && isFinite(v)`